promql/quantile.go

```diff
@@ -328,7 +342,7 @@ func histogramQuantile(q float64, h *histogram.FloatHistogram) float64 {
 // If lower or upper is NaN, NaN is returned.
 //
 // If lower >= upper and the histogram has at least 1 observation, zero is returned.
-func histogramFraction(lower, upper float64, h *histogram.FloatHistogram) float64 {
+func HistogramFraction(lower, upper float64, h *histogram.FloatHistogram) float64 {
```

Comment:
Add to the doc comment something like: "HistogramFraction is exported as it may be used by other PromQL engine implementations."